docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -19,15 +19,15 @@ This document guides a developer or administrator through the steps necessary to
 1. Choose [Db2](https://cloud.ibm.com/catalog/services/db2).
 
 1. Select the Pricing Plan:
-    - The IBM FHIR Server recommends the Flex plan for a production workload, development and experiments.
+    - The IBM FHIR Server recommends the *Standard* plan for a production workload, development and experiments.
```

Comment:
We don't have experience with the *Enterprise* plan yet, but I'm not sure if we should exclude it.
Maybe just focus on NOT using the free plan for now?

Reply:
Fixed